appserver/tests/payara-samples/test-utils/src/main/java/fish/payara/samples/CliCommands.java

```diff
@@ -33,19 +33,18 @@ public static int payaraGlassFish(List<String> cliCommands) {
     }
 
     public static int payaraGlassFish(List<String> cliCommands, List<String> output) {
-
```

Comment:
This will work only on the server side, but not on client junit test side. But if you would set the com.sun.aas.productRoot JVM option also for surefire, it wouold be better.

Reply:
correct. Ecosystem is already assuming this